src/main/java/edu/kit/datamanager/pit/pidgeneration/generators/PidSuffixGenUpperCase.java

```diff
@@ -0,0 +1,23 @@
+package edu.kit.datamanager.pit.pidgeneration.generators;
+
+import edu.kit.datamanager.pit.pidgeneration.PidSuffix;
+import edu.kit.datamanager.pit.pidgeneration.PidSuffixGenerator;
+
+/**
+ * Generates a PID suffix based on a contained generator and returns the result
+ * in lower case.
```

Comment:
should be "upper case" (also in `PidSuffixGenPrefixed.java`)

Reply:
Thank you :) fixed.